internal/resources/templates/login.go

```diff
@@ -12,14 +12,15 @@ var Login = `
 								Sign In using your GIN credentials
 							</h3>
 							<div class="ui attached segment">
```

Comment:
Please move the error message `div` further up, to directly after the `<div class="ui attached segment">`.